dev-python/grpcio/grpcio-1.66.2.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="High-performance RPC framework (python libraries)"
+HOMEPAGE="https://grpc.io"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
```

Comment:
Did you actually test it on each of these architectures?

Reply:
> Did you actually test it on each of these architectures?

Only on `amd64` and `arm64`. I can't check on x86, but I can't just delete it, there will be pkgcheck errors.

Reply:
Delete it from all packages (including new version of `google-api-core`), and we'll readd it after testing.